config.toml

```diff
@@ -577,13 +577,18 @@ humanoid or AI!**
 Our "Code of Conduct" is **"Be Excellent to Each Other"** AKA the Golden Rule.
 Failing that, it is "Do not be an Ass* or we will kick your ass out!".
 
-Asking questions of a speaker during their talk, to get clarity or debate a
+Asking questions of a speaker during Q&A time to get clarity or debate a
 point is NOT being an ass – heckling or haranguing the speaker IS. If you are
 not sure, ask, or err on the side of basic decency and common courtesy. If what
 they are doing would not be acceptable to have done to you, your best friend,
 your worst enemy, your sister, niece, daughter, brother, nephew, son, mother,
 father, or any human being, do not let them treat anyone else that way – whether
- you know them or not. **If someone asks you to stop – stop.**
+you know them or not. **If someone asks you to stop – stop.**
+
+BSides Atlanta seeks to ensure no attendee is harassed. This includes, but is
```

Comment:
Are we BSidesATL or BSides Atlanta? We should be consistent.

Reply:
We've been using `BSides Atlanta` in text and `BSidesATL` as slugs and usernames. The site header has the long-form version, so I went with that here.